packages/react-components/review/beta/react-components.api.md

```diff
@@ -64,6 +64,38 @@ export type AnnouncerProps = {
     ariaLive: 'off' | 'polite' | 'assertive' | undefined;
 };
 
+// @beta
+export interface AtMentionDisplayOptions {
+    atMentionSuggestionRenderer?: (suggestion: AtMentionSuggestion) => JSX.Element;
+}
+
+// @internal
```

Comment:
Small tip: it's okay to have @internal here in the react-components.api.md, the one we really care is the final mega package